features/checkout/ability_to_checkout_as_guest_with_a_registered_email.feature

```diff
@@ -11,7 +11,7 @@ Feature: Checking out as guest with a registered email
         And the store allows paying offline
         And there is a customer account "john@example.com"
 
-    @ui @api
```

Comment:
This step is temporarily disabled and will be fixed in next iteration, We have secured the customers cart against operating by unauthorized visitor. But now user that has account in shop and try to buy something without login can't do this. Blocked by query in `OrderGetMethodItemExtension.php` and `OrderMethodsItemExtension.php` , As a visitor the query allow to get or to set order for situation when: `customer == null` or `customer.user == null`, but in this specific case should allow to continue shops when `customer.user == certainUser`, Possibility solution: create context that will be store customer id after addressing step.